src/analysis/retail/paladin/holy/normalizers/CastLinkNormalizer.ts

```diff
@@ -27,35 +35,152 @@ const EVENT_LINKS: EventLink[] = [
     },
   },
   {
-    linkRelation: DAYBREAK_HEALING,
+    linkRelation: DAYBREAK_MANA,
     linkingEventId: TALENTS.DAYBREAK_TALENT.id,
     linkingEventType: EventType.Cast,
-    referencedEventId: SPELLS.GLIMMER_OF_LIGHT_HEAL_TALENT.id,
-    referencedEventType: [EventType.Heal, EventType.HealAbsorbed],
-    forwardBufferMs: 350,
+    referencedEventId: SPELLS.DAYBREAK_ENERGIZE.id,
+    referencedEventType: [EventType.ResourceChange],
     anyTarget: true,
     reverseLinkRelation: FROM_DAYBREAK,
+    forwardBufferMs: 100,
```

Comment:
i can do this... but i don't think it's made the code any better. i don't know how server events and/or combat log events are batched. so I don't have any reason to believe there's a relationship between the interval between a daybreak cast to a daybreak energize and the interval between a holy shock cast and a glimmer heal -- which makes it unproductive for those configuration values to share a constant. Since e.g. GLIMMER_PROC links don't share a buffer and HOLY_SHOCK_SOURCE links don't share a buffer, i can't even think of semantic names so all i've really got here are some empirical observations about some particular link->reference relations happening within a short/medium/long kind of timeframe...

Reply:
even here, i've accidentally stumbled on some "bad" language... what I'm really getting at is why do the 5 holy shocks from a divine toll seem to happen within MED_BUFFER_MS while the 2 holy shocks from rising sunlight can take up to LONG_BUFFER_MS? We don't really know and we don't really have a way to know. If we observe that the gaps between two types of events are similar, is it meaningful or coincidence?